src/css/base.styl

```diff
@@ -1674,10 +1674,41 @@ nav ul
         .text
           height: 195px
 
+    // Card implementation div
+    .implementation
+      width: 150px
```

Comment:
To deal with the width issues, couldn't you just change this to `max-width` to make it automatically fit to the contents?